blog/2020-12-09-eclipse-vert-x-4-released.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Eclipse Vert.x 4 released!
+category: releases
+authors:
+  - name: Julien Viet
+    github_id: vietj
+summary: >-
+  Vert.x 4 has just been released and it's the best thing since sliced buffers.
+---
+
+We are extremely pleased to announce the release of Vert.x 4.
+
+## What's new in Vert.x 4?
+
+Vert.x 4 is the evolution of the Vert.x 3.x series that brings [key features](https://vertx.io/blog/whats-new-in-vert-x-4/) to Vert.x.
```

Comment:
Another sentence with the highlights (like just the paragraph names of the post you linked)?

Reply:
I think it is better here to simply link to the page that does this job and not extend inline